TCL/polarDensity_for_DTA.tcl

```diff
@@ -558,18 +496,26 @@ proc polarDensityBin { config_file_name } {
             Align "occupancy $helixlist and $backbone_selstr"
         }
         ;# outputs protein positions
-        Protein_Position $chainlist $helixlist $backbone_selstr $midplane_selstr
+        output_helix_centers $chainlist $helixlist $backbone_selstr $midplane_selstr
         ;# initialize some constants
         set area [get_avg_area top]
-        set nframes [molinfo top get numframes]
+
+        if { $start_frame > $nframes } {
+            puts "Error: specified start frame $start_frame is greater than number of frames $nframes" 
+            set end $nframes
+        }
+        if { $end_frame > $nframes } {
+            puts "Warning: specified end frame $end_frame is greater than number of frames; setting end frame to $nframes" 
+            set end $nframes
+        }
         $sel delete
         puts "Acyl Chain:\t$species"
         set low_f [open "${outfile}.low.dat" w]
         set upp_f [open "${outfile}.upp.dat" w]
         set low_f_avg [open "${outfile}.low.avg.dat" w]
         set upp_f_avg [open "${outfile}.upp.avg.dat" w]
         set dtheta [expr 360.0/(1.0*($Ntheta))]
-        set totals [get_leaflet_totals "resname $species" $headname $tailname $lipidbeads_selstr 0 $LEAFLET_SORTING_ALGORITHM ]
+        set totals [frame_leaflet_assignment "resname $species" $headname $tailname $lipidbeads_selstr $start_frame $start_frame $LEAFLET_SORTING_ALGORITHM ]
```

Comment:
Is this meant to go from start_frame to start_frame? Or should the second one be end_frame?

Reply:
This is correct; the intended behavior here is to just run over the first frame.